patches/spirv/0001-update-SPIR-V-headers-for-SPV_INTEL_split_barrier.patch

```diff
@@ -0,0 +1,43 @@
+From 05dbb133140653f21efc1feebfcb614386cc2a62 Mon Sep 17 00:00:00 2001
+From: Feng Zou <feng.zou@intel.com>
+Date: Fri, 25 Feb 2022 13:51:07 +0800
+Subject: [PATCH] update SPIR-V headers for SPV_INTEL_split_barrier
+
+PR:https://github.com/KhronosGroup/SPIRV-Headers/pull/268
+---
+ lib/SPIRV/libSPIRV/spirv.hpp | 5 +++++
+ 1 file changed, 5 insertions(+)
+
+diff --git a/lib/SPIRV/libSPIRV/spirv.hpp b/lib/SPIRV/libSPIRV/spirv.hpp
+index cbe5a94c..631396b8 100644
+--- a/lib/SPIRV/libSPIRV/spirv.hpp
++++ b/lib/SPIRV/libSPIRV/spirv.hpp
+@@ -983,6 +983,7 @@ enum Capability {
+   CapabilityAtomicFloat64AddEXT = 6034,
+   CapabilityFastCompositeINTEL = 6093,
+   CapabilityOptNoneINTEL = 6094,
++  CapabilitySplitBarrierINTEL = 6141,
+   CapabilityMax = 0x7fffffff,
+ };
+ 
+@@ -1538,6 +1539,8 @@ enum Op {
+   OpFPGARegINTEL = 5949,
+   OpAtomicFAddEXT = 6035,
+   OpTypeBufferSurfaceINTEL = 6086,
++  OpControlBarrierArriveINTEL = 6142,
++  OpControlBarrierWaitINTEL = 6143,
+   OpMax = 0x7fffffff,
+ };
+ 
+@@ -2076,6 +2079,8 @@ inline void HasResultAndType(Op opcode, bool *hasResult, bool *hasResultType) {
+       *hasResult = true;
+       *hasResultType = false;
+       break;
++    case SpvOpControlBarrierArriveINTEL: *hasResult = false; *hasResultType = false; break;
++    case SpvOpControlBarrierWaitINTEL: *hasResult = false; *hasResultType = false; break;
```

Comment:
is this clang-formatted?

Reply:
No. Most of case statements are in one line, except the one in line 2078 ~ 2081. So I follow the code format in most of lines.

Reply:
ok